src/account_multisig.ts

```diff
@@ -110,7 +111,7 @@ export class AccountMultisig extends Account {
                 await super.signAndSendTransaction(this.accountId, 
                     [functionCall('delete_request', { request_id: requestIdToDelete }, MULTISIG_GAS, MULTISIG_DEPOSIT)]);
             } catch(e) {
-                console.warn('Attempt to delete an earlier request before 15 minutes failed. Will try again.');
+                getLogger().debug('Attempt to delete an earlier request before 15 minutes failed. Will try again.');
```

Comment:
Looks like we should check `errorType` here :/ Not a blocker for this PR.
Why `debug` instead of `warn`?

Reply:
i think you're right.